src/common/webviewEvents.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-export type WebviewEvent = "getState" | "getUrl" | "openInEditor" | "ready" | "setState" | "telemetry" | "websocket";
+export type WebviewEvent = "getState" | "getUrl" | "openInEditor" | "ready" | "setState" | "telemetry" | "websocket" |
+                            "close";

```

Comment:
This should only be used for messages that go between the devtools and the extension. 'close' is just being emitted on the extension side only in response to the real websocket closing down.
Instead of adding this here to fix the 'listens for all the emit events' test, you should update the test to just check that the number of events it has hooked is +1 // for the close event. And then add a check that ```expect(hookedEvents).toContain("close")```